web/src/abstreet/abstreet.ts

```diff
@@ -0,0 +1,20 @@
+import * as wasm_pkg from "./wasm_pkg/game.js";
```

Comment:
I believe on the call the other day, you explained how this boilerplate is not reasonably avoidable, right?

Reply:
I don't know how to reasonably avoid it - it might be possible though.

Reply:
I think it's fine as it is -- we won't be splitting out new binaries often, and this is pretty easy to copy over.